etcdmain/config.go

```diff
@@ -317,6 +356,141 @@ func (cfg *config) Parse(arguments []string) error {
 		}
 	}
 
+	return nil
+}
+
+func (cfg *config) configFromFile() error {
+	// Read in the contents of the config file.
```

Comment:
remove the comments? i think the code itself is pretty clear about what is doing actually